### PR TITLE
translate(mancala): setup i18n and translate to PT

### PR DESCRIPTION
**Pull request by leocaseiro**
Migrate from Enzyme to rtl for mancala

#### Checklist

* [x] Use a separate branch in your local repo (not `master`).
* [x] I've read and agree with the [contribution guidelines](https://www.freeboardgames.org/docs/?path=/story/documentation-contributing--page).

### CODE REVIEW

File: web/src/games/mancala/Shapes.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ const SeedCount = (props: IShapeProps) => (
 export const Hole = (props: IShapeProps) => (
   <g>
     <circle
+      data-testid="hole"
```
Comment (leocaseiro) [inline web/src/games/mancala/Shapes.tsx:30]: `react-testing-library`

File: web/src/games/mancala/board.test.tsx [hunk 2]
```diff
@@ -10,14 +10,16 @@ describe('mancala', () => {
...
+
+    const render = makeRender({ gameCode: 'mancala' });
+    render(
       <Board
         G={state0.G}
         ctx={state0.ctx}
         moves={client.moves}
         playerID={'0'}
         gameArgs={{
-          gameCode: 'zooparade',
```
Comment (leocaseiro) [inline web/src/games/mancala/board.test.tsx:20]: not really :D

File: web/src/games/mancala/game.ts [hunk 3]
```diff
@@ -15,7 +15,7 @@ export interface IG {
   playerStoreCount: Array<number>;
 }
 
-export const MancalaGame = {
+export const MancalaGame: Game<IG> = {
```
Comment (leocaseiro) [inline web/src/games/mancala/game.ts:18]: TS ❤️

File: web/src/games/reversi/locales/pt.json [hunk 4]
```diff
@@ -17,7 +17,7 @@
     "green": "Verde",
     "players_turn": "Vez do {{ playerName }}",
     "place_piece": "Coloca peça",
-    "waiting_for_opponent": "Esperando por adiversário",
+    "waiting_for_opponent": "Esperando por adversário",
```
Comment (leocaseiro) [inline web/src/games/reversi/locales/pt.json:20]: noticed the typo

File: web/src/games/mancala/board.test.tsx (hunk 2 above)
Comment (vdfdev) [reply web/src/games/mancala/board.test.tsx:20]: Lol

File: web/src/games/reversi/locales/pt.json (hunk 4 above)
Comment (vdfdev) [reply web/src/games/reversi/locales/pt.json:20]: Someone (me) needs to go back to primary school :P

**Review by vdfdev — APPROVED** [review]

**Comment by vdfdev** [comment]
CI is failing :(

File: web/src/games/reversi/locales/pt.json (hunk 4 above)
Comment (leocaseiro) [reply web/src/games/reversi/locales/pt.json:20]: It was me, actually

**Comment by leocaseiro** [comment]
I improved types, and I think I found a bug in this game with some deprecated code.
It seems to be working fine now. I'm not 100% familiar with the rules, but I know you can move it again if you get some type of score.

@vdfdev, do you want to confirm with the author, do you know this game well?

cc @pestopancake

**Comment by vdfdev** [comment]
@leocaseiro I know the basic of the rules, that you can play again if the last piece ends in the header or footer of the board. I think this should be tested by unit tests ? I trust your judgement here, you can merge.

**Comment by leocaseiro** [comment]
Cool, that is working as expected. MERGED :D

**Pull request merged**